### PR TITLE
Travis shellcheck orig

### PR DESCRIPTION
**Pull request by rkhozinov**

### CODE REVIEW

File: servers/partner-ci/builders/build-job-download-iso.sh [hunk 1]
```diff
@@ -1,25 +1,12 @@
 #!/bin/bash -xe
-#[[ $RELEASE != '9.0-mos' ]] && { exit 1; } 
+#[[ $RELEASE != '9.0-mos' ]] && { exit 1; }
```
Comment (naumvd95) [inline servers/partner-ci/builders/build-job-download-iso.sh:2]: why we need this raw?

File: servers/partner-ci/builders/build-plugin-manila.sh [hunk 2]
```diff
@@ -5,19 +5,19 @@ git log  --pretty=oneline | head
...
+  cmd2="(find $PUPPETLINT_PATH -name '*.pp'  ! -name $PUPPETLINT_IGNORE -print0 )"
+  cmd3="(find $PUPPETLINT_PATH -name '*.pp'  ! -name $PUPPETLINT_IGNORE -print0 )"
 fi
 
-$cmd1 | xargs -0 -P1 -L1 -I '%' erb -P -x -T '-' % | ruby -c
-$cmd2 | xargs -0 -P1 -L1 puppet parser validate --verbose
-$cmd3 | xargs -0 -r -P1 -L1 puppet-lint \
+"${cmd1[@]}" | xargs -0 -P1 -L1 -I '%' erb -P -x -T '-' % | ruby -c
+"${cmd2[@]}" | xargs -0 -P1 -L1 puppet parser validate --verbose
+"${cmd3[@]}" | xargs -0 -r -P1 -L1 puppet-lint \
```
Comment (naumvd95) [inline servers/partner-ci/builders/build-plugin-manila.sh:20]:
```string=abcABC123ABCabc
echo ${string[@]}               # abcABC123ABCabc
echo ${string[*]}               # abcABC123ABCabc 
echo ${string[0]}               # abcABC123ABCabc
echo ${string[1]}               # No output!
                               # Why?
echo ${#string[@]}              # 1
                                # One element in the array.
                                # The string itself.
```

File: servers/partner-ci/builders/common.prepare.sh [hunk 3]
```diff
@@ -160,8 +160,8 @@ function dospy_list {
   prefix=$1
   dos.py sync
   [ -z $prefix ] && \
-    echo $(dos.py list | tail -n +3) || \
-    echo $(dos.py list | tail -n +3  | grep $prefix)
+    dos.py list | tail -n +3 || \
```
Comment (naumvd95) [inline servers/partner-ci/builders/common.prepare.sh:163]: sure?
please attach checker-number

File: servers/partner-ci/builders/contrail.test.sh [hunk 4]
```diff
@@ -154,11 +151,11 @@ export PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${WORKSPACE}:${PLUGIN_WORKSPACE}
...
 export SYSTEST_PID=$!
-	
-if [[ $VCENTER_USE == "true"  ]]; then
+
+if [[ "$VCENTER_USE" == "true"  ]]; then
   #Wait before environment is created
-  while [ true ]; do
-    [ $(virsh net-list | grep $ENV_NAME | wc -l) -eq 5 ] && break || sleep 10
+  while true ; do
+    [ $(virsh net-list | grep -c "$ENV_NAME" )  -eq 5 ] && break || sleep 10
```
Comment (naumvd95) [inline servers/partner-ci/builders/contrail.test.sh:158]: replace wc with grep -c option?
did u check this?

File: servers/partner-ci/builders/dvs.test.sh [hunk 5]
```diff
@@ -28,11 +26,9 @@ if [ "${SNAPSHOTS_ID}" != "released" ]; then
 fi
 
 if [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]]; then
-  export SNAPSHOTS_ID=$(cat snapshots.params | grep -Po '#\K[^ ]+')
+  export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params)
 fi
```
Comment (naumvd95) [inline servers/partner-ci/builders/dvs.test.sh]: seems now there is now snap_id is empty - checker

File: servers/partner-ci/builders/vcenter.test.sh [hunk 6]
```diff
@@ -27,14 +27,13 @@ if [ "${SNAPSHOTS_ID}" != "released" ]; then
...
 fi
 
-if [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]]; then
-  export SNAPSHOTS_ID=$(cat snapshots.params | grep -Po '#\K[^ ]+')
-fi
-
-[ -z "${SNAPSHOTS_ID}" ] && { echo SNAPSHOTS_ID is empty; exit 1; }
+[[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]] && \
+  id=$(grep -Po '#\K[^ ]+' < snapshots.params)
+  export SNAPSHOTS_ID=$id
```
Comment (naumvd95) [inline servers/partner-ci/builders/vcenter.test.sh:32]: export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params )
seems like this?

File: servers/partner-ci/builders/vcenter.test.sh [hunk 7]
```diff
@@ -27,14 +27,13 @@ if [ "${SNAPSHOTS_ID}" != "released" ]; then
...
 
 if [ -f build.plugin_version ]; then
-  export DVS_PLUGIN_VERSION=$(grep "PLUGIN_VERSION" < build.plugin_version | cut -d= -f2 )
+  version=$(grep "PLUGIN_VERSION" < build.plugin_version | cut -d= -f2 )
+  export DVS_PLUGIN_VERSION=$version
```
Comment (naumvd95) [inline servers/partner-ci/builders/vcenter.test.sh:36]: the same as above

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/builders/contrail.test.sh (hunk 4 above)
Comment (naumvd95) [reply servers/partner-ci/builders/contrail.test.sh:158]: image

Comment (naumvd95) [reply servers/partner-ci/builders/contrail.test.sh:158]: checked

File: servers/partner-ci/builders/build-plugin-manila.sh (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/build-plugin-manila.sh:20]: ${#string[@]} # elements count

File: servers/partner-ci/builders/contrail.test.sh (hunk 4 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/contrail.test.sh:158]: good

File: servers/partner-ci/builders/common.prepare.sh (hunk 3 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/common.prepare.sh:163]:
```
dos.py list      # output list, but could be saved to a var
$(dos.py list) # output list, but could be save to a var
```

File: servers/partner-ci/builders/dvs.test.sh (hunk 5 above)
Comment (rkhozinov) [inline servers/partner-ci/builders/dvs.test.sh]: I don't understand

File: servers/partner-ci/builders/vcenter.test.sh (hunk 6 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/vcenter.test.sh:32]: shellcheck's warning

File: servers/partner-ci/builders/vcenter.test.sh (hunk 7 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/vcenter.test.sh:36]: the same as above

**Review by naumvd95 — APPROVED** [review]

**Pull request closed without merging**